Step2

ArkHak opened the pull request:
Homework for Lesson 2

ArkHak committed:
ver 0.1.1
***Lesson_2***
-добавлена навигация NavCompose.
-убраны незадействоанные иконки меню.
-refactoring.
ver 0.1.5
***Lesson_2***
-Созданы стили для почти всех view.
-Созданы дополнительные темы приложения.
-Реализован переключатель тем в настройках.
-Для примера добавлен шрифт.
-Темная тема по умолчанию.
-refactoring.

ArkHak committed:
ver 0.1.51
-refactoring.

ArkHak committed:
ver 0.1.1
***Lesson_2***
-добавлена навигация NavCompose.
-убраны незадействоанные иконки меню.
-refactoring.

ArkHak committed:
Merge remote-tracking branch 'origin/step2' into step2

ArkHak committed:
ver 0.2.3
***Lesson_3***
-добавлена, настроена и кастомизирована BottomNavigationView.
-на разных экранах - разный функционал.

ArkHak committed:
ver 0.2.5
***Lesson_3***
-добавлен ViewPager2 с TabLayout + анимация переслистывания

ArkHak committed:
ver 0.2.6
***Lesson_3***
-добавлен placeholder загрузки в POD;
-закруглены края у bottom_sheet и Image_pod

Zauberer commented on app/src/main/java/com/example/appnasapi/PODSliderFragment.kt:
```diff
@@ -0,0 +1,90 @@
+                val pageWidth = width
+                val pageHeight = height
+                when {
+                    position < -1 -> {
```
-1 и 1 - нужно осмысленные константы делать. Заведите именованые константы

ArkHak committed:
ver 0.2.7
***Lesson_3***
-решена проблема с NightTheme - теперь app работает;
-корректная загрузка POD;

ArkHak committed:
ver 0.3.4
***Lesson_4***
-MotionLayout для Википедии
-анимация слайдера

ArkHak committed:
ver 0.4.1
***Lesson_5***
-добавлена анимация увеличения изображения при клике
